bin/improver-nbhood

```diff
@@ -63,6 +63,11 @@ def main():
                        'should be used. For example: 10,12,14 1,2,3 '
                        'where a lead time of 1 hour uses a radius of 10km, '
                        'a lead time of 2 hours uses a radius of 12km, etc.')
+    parser.add_argument('--ens_factor', metavar='ENS_FACTOR', type=float,
+                        default=1.0, required=False,
```

Comment:
Yes argparse assumes all flag arguments are optional but since we have flag arguments for radii and they are not optional (either one or the other) I thought I would make it obvious that this is an optional argument. However I have now removed this.